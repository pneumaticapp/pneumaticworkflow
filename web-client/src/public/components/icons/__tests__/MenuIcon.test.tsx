/* eslint-disable */
/* prettier-ignore */
import * as React from 'react';
import { shallow } from 'enzyme';

import { MenuIcon } from '../MenuIcon';

describe('Menu Icon', () => {
  it('корректно рендерится', () => {
    const wrapper = shallow(<MenuIcon />);

    expect(wrapper).toMatchSnapshot();
  });
});
