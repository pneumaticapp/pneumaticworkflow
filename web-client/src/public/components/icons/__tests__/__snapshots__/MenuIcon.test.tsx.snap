// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Icon корректно рендерится 1`] = `
<Fragment>
  <svg
    className="main"
    viewBox="0 0 9 17"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      height="1"
      width="7"
      x="0.48"
      y="0.5"
    />
    <rect
      height="1"
      width="7"
      x="0.48"
      y="7.5"
    />
    <rect
      height="1"
      width="7"
      x="0.48"
      y="15.5"
    />
  </svg>
  <svg
    className="sub"
    viewBox="0 0 18 17"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      height="1"
      width="16"
      x="1.56"
      y="0.5"
    />
    <rect
      height="1"
      width="16"
      x="1.56"
      y="7.5"
    />
    <rect
      height="1"
      width="16"
      x="1.56"
      y="15.5"
    />
  </svg>
</Fragment>
`;
